src/stats.js

```diff
@@ -27,12 +27,19 @@ const self = (module.exports = {
         })
     );
 
-    const result = {}
-
-    ["readyForReview", "firstReview", "firstApprovedReview", "merged"].forEach(column => result[column] = {
-      fromPrCreation: averageValues(statsResult.map(sr => sr[column].fromPrCreation)),
-      fromPreviousStep: averageValues(statsResult.map(sr => sr[column].fromPreviousStep))
-    })
+    const result = {}[
```

Comment:
This doesn't look right.

I think that Prettier merged two statements into one. We should add a `;` after the `{}` to make it clear

```suggestion
    const result = {};
    
    [
```

Reply:
👍 will update in the PR 🙏🏻